### PR TITLE
Move Background Concepts and Example Scenario into the introduction

### PR DESCRIPTION
**Pull request by steven1046**

### CODE REVIEW

File: json-guides/circuit-breaker.json [hunk 1]
```diff
@@ -11,63 +11,63 @@
...
+              "You'll start with a sample bank scenario and see how it fails when no fault tolerance feature is in the example code. You'll then add the <code>mpFaultTolerance-1.0</code> feature to the server.xml file to enable you to use the MicroProfile fault tolerance feature. Next, you'll include the <code>@CircuitBreaker</code> annotation so that your microservice fails immediately to prevent repeated calls that are likely to fail.  You will explore and modify the parameters in the <code>@CircuitBreaker</code> annotation and simulate successful or failed requests to the microservice to see how the circuit state changes.  Finally, you will add a fallback service to see how an alternate method for a failing service can be invoked.",
+              "We are using the MicroProfile <b><i>CircuitBreaker</i></b> and <b><i>Fallback</i></b> policies for our fault tolerance implementation.<br>"
```
Comment (lavenal) [inline json-guides/circuit-breaker.json]: just want to confirm that this new sentence is supposed to be there.

File: json-guides/circuit-breaker.json [hunk 2]
```diff
@@ -11,63 +11,63 @@
...
-            "content": [
+            "sections" : [
               {
-                "displayType":"pod",
-                "content": "<div class='bankScenarioImg'><img src='/guides/iguide-circuit-breaker/html/interactive-guides/circuit-breaker/images/intro.svg' alt='Online Banking Microservices'></div>"
+                "name": "backgroundConcepts",
+                "title": "Background Concepts",
+                "description": [
+                  "<b><i>Microservice</i></b> is an architecture that structures an application loosely from other services so it is independent and modular.<br>",
+                  "<h4><b><u>Circuit Breaker</u></b></h4>",
```
Comment (lavenal) [inline json-guides/circuit-breaker.json]: what happened to all the images for the circuit breaker?

Comment (steven1046) [reply json-guides/circuit-breaker.json]: I think something got overridden, i will check today.

File: json-guides/circuit-breaker.json (hunk 1 above)
Comment (steven1046) [reply json-guides/circuit-breaker.json]: Moved it back and put the images back in. I will wait to deliver until Monday so you can check to make sure I didn't override anything else you did.

**Pull request merged**